### DISCUSSION TITLE
Getting Confidence Score of Tesseract-ocr

### DISCUSSION THREAD

**Discussion by Yukititit** [pymupdf/PyMuPDF#2158] [Q&A] [answered]
I am currently working on a application with Text Extraction feature and just start using PyMuPDF. It works surprisingly good with very little effort on development. 

However, is there a way for me to also get the confidence score from tesseract when using the get_textpage_ocr function?

**Comment by JorjMcKie** [comment, marked as answer]
No, sorry there is no access to this value.
